Tests/AssemblyWithInheritanceTests.cs

```diff
@@ -20,11 +24,18 @@ static AssemblyWithInheritanceTests()
             weavingTask.ExecuteTestRun("AssemblyWithExternalInheritance.dll")
         };
 
-        // Must use Assembly.LoadFrom, else the non-woven assemblies will be loaded as dependencies!
+#if NETFRAMEWORK
         assemblies = testResults
-            .Select(result => result.AssemblyPath)
-            .Select(path => Assembly.LoadFrom(path))
+            // Must use Assembly.LoadFrom, else the non-woven assemblies will be loaded as dependencies!
+            .Select(result => Assembly.LoadFrom(result.AssemblyPath))
             .ToArray();
+#else
+        var folder = Path.GetDirectoryName(testResults.First().AssemblyPath);
+        var loadContext = new PluginLoadContext(folder);
```

Comment:
You can replace this with `var loadContext = new AssemblyLoadContext(null);` and remove the `PluginLoadContext` class entirely since you're loading the assemblies in the "correct" order.

The `AssemblyDependencyResolver` doesn't help since you have no `.deps.json` file in the `fodytemp` directory.